confdb/confdb.go

```diff
@@ -1080,9 +1080,8 @@ func getValuesThroughPathsImpl(storagePath string, unmatchedSuffix []accessor, v
 				for path, val := range pathsToValues {
 					storagePathsToValues[path] = val
 				}
-
-				storagePathsToValues[newStoragePath] = el
```

Comment:
is this a fix of some sort?

Reply:
It is, it was a left over caught by one of the new tests.